wiki/en/misc/footertext.md

```diff
@@ -1,6 +0,0 @@
-
```

Comment:
Why is this removed?

Reply:
If that's the same thing as the "wiki" footer, then see above (I get confused with all the different files controlling different things).